cost/turbonomics/credential_refresh/turbonomic_cred_refresh.pt

```diff
@@ -0,0 +1,166 @@
+name "Turbonomic Credential Refresh"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that refreshes the cookie used to authenticate with Turbonomic APIs"
+severity "low"
+category "Operational"
+default_frequency "hourly"
+info(
+  version: "0.1",
+  provider: "",
+  service: "",
+  policy_set: "",
+	publish: "false"
+)
+
+parameter "turbo_username" do
+  type "string"
+  label "Turbonomic Username"
+  description "The username used to authenticate with Turbonomic APIs"
+end
+
+parameter "turbo_password" do
+  type "string"
+  no_echo true
+  label "Turbonomic Password"
+  description "The password used to authenticate with Turbonomic APIs"
+end
+
+parameter "turbo_host" do
+  type "string"
+  label "Turbonomic Host"
+  description "Your Turbonomic host or IP."
+end
+
+###############################################################################
+# Authentication
+###############################################################################
```

Comment:
Most of the time Flexera people leave a blank line after a section comment, I would suggest following that same approach.

```suggestion
###############################################################################

```